src/encoding.ts

```diff
@@ -75,6 +75,11 @@ export interface Encoding {
    * stack order for stacked marks or order of data points in line marks.
    */
   order?: OrderFieldDef | OrderFieldDef[];
+
+  /**
+   * Tooltip adds basic string tooltip support.
```

Comment:
"The tooltip text to show upon mouse hover."  would be a bit better.  

(FYI, we use this JSDocs comment in the JSON schema, which is in turn used to populate the docs.)